src/main/java/mate/academy/internetshop/Application.java

```diff
@@ -21,7 +25,17 @@ public static void main(String[] args) {
         System.out.println(productService.get(product.getId()));
         System.out.println(Storage.products);
 
-        productService.delete(product.getId());
-        System.out.println(Storage.products);
+        User user = new User("Shion");
+        UserService userService = (UserService) injector.getInstance(UserService.class);
+        System.out.println(userService.create(user));
+
+        user.setName("Benio");
+        userService.update(user);
+        System.out.println(userService.get(user.getId()));
+        System.out.println(userService.delete(user.getId()));
+
+        ShoppingCart shoppingCart = new ShoppingCart();
+        ShoppingCartService shoppingCartService
```

Comment:
provide more examples of ShoppingCartService usage

Reply:
@boroda4436 
Hi!

Additional examples and fixes added :)